### PR TITLE
test ci-workflow

### PR DESCRIPTION
**Pull request by thientuquangtrung**
new feature has been done, pls test ci workflow

### CODE REVIEW

File: customer/src/services/customer-service.test.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ describe("Customer service", () => {
     describe("Sign in function", () => {
         // which case we are testing
```
Comment (thientuquangtrung) [inline customer/src/services/customer-service.test.js:5]: is it necessary?

**Review by thientuquangtrung — COMMENTED** [review]
pls check this again!!!

File: customer/src/services/customer-service.test.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ describe("Customer service", () => {
...
         test("should user input properly", () => {});
+        // test CI work flow 
+        // test fix
```
Comment (thientuquangtrung) [inline customer/src/services/customer-service.test.js:8]: great!

**Review by thientuquangtrung — COMMENTED** [review]
good job!

**Review by trungntq19 — APPROVED** [review]
Good job bro!

**Pull request merged**